tex/bib.bib

```diff
@@ -560,6 +587,20 @@ @ARTICLE{Lacis1972
 adsnote = {Provided by the SAO/NASA Astrophysics Data System}
 }
 
+@article{Livengood2011,
```

Comment:
@ericagol Can you send me a PDF of this article? I don't have institutional access to AsBio. Thanks!